src/handlers/test/interventions-processor-lambda.test.ts

```diff
@@ -238,5 +238,41 @@ describe('intervention processor handler', () => {
         batchItemFailures: [],
       });
     });
+
+    it('should ignore if level of confidence is not P2', async () => {
+      eventValidationMock.mockReturnValueOnce(void 0);
+      mockRecord = {
+        messageId: '123',
+        receiptHandle: '',
+        body: JSON.stringify({
+          timestamp: getCurrentTimestamp().milliseconds + 500_000,
+          user: {
+            user_id: 'abc',
+          },
+          event_name: 'IPV_IDENTITY_ISSUED',
+          extensions: {
+            levelOfConfidence: "P3",
```

Comment:
can we make this P1 so that if levels of confidence changes and the add P3 then in the future we will check if >= P2